great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -453,90 +447,44 @@ def validation_definitions(self) -> ValidationDefinitionFactory:
         return self._validation_definitions
 
     @property
-    def expectations_store_name(self) -> Optional[str]:
-        return self.variables.expectations_store_name
-
-    @expectations_store_name.setter
-    @public_api
-    @new_method_or_class(version="0.17.2")
-    def expectations_store_name(self, value: str) -> None:
-        """Set the name of the expectations store.
-
-        Args:
-            value: New value for the expectations store name.
-        """
-
-        self.variables.expectations_store_name = value
-        self._save_project_config()
+    def expectations_store_name(self) -> str:
+        return StoreManager.expectations_store_name
```

Comment:
I don't like this but a few components still rely on name - we should remove these but stopping user configuration is a solid first step